libraries/chain/include/eosio/chain/types.hpp

```diff
@@ -405,7 +405,10 @@ namespace eosio::chain {
    // http_plugin thread pool) and which completes the API processing and returns the result T.
    // -------------------------------------------------------------------------------------------------------
    template<typename T>
-   using next_function_variant = std::variant<fc::exception_ptr, T, std::function<T()>>;
+   using t_or_exception = std::variant<T, fc::exception_ptr>;
```

Comment:
Can a more descriptive name be chosen for `t_or_exception`?

Reply:
I can't think of one. Do you have any suggestion?

Reply:
last_function or something? Hard to come up with a good name😀

Reply:
I like `t_or_exception` over `last_function_variant`.

Reply:
OK, I like it too, it really says that it is either a `T` or an `exception`.